src/sage/schemes/toric/fano_variety.py

```diff
@@ -21,8 +21,8 @@
 of Calabi-Yau manifolds via constructions due to Victor V. Batyrev
 [Bat1994]_ and Lev A. Borisov [Bor1993]_.
 
-From the combinatorial point of view "crepant" requirement is much more simple
-and natural to work with than "coherent." For this reason, the code in this
+From the combinatorial point of view, the "crepant" requirement is much simpler
+and more natural to work with than "coherent." For this reason, the code in this
```

Comment:
```suggestion
From the combinatorial point of view, the "crepant" requirement is much more simple
and natural to work with than "coherent." For this reason, the code in this
```
The rest is a bit better English IMO. The rest of your changes look good.